governance.md

```diff
@@ -0,0 +1,114 @@
+| Ralph Bean      | Red Hat | [ralphbean](https://github.com/ralphbean)   | May 2024 | May 2025 |
+| Andrew McNamara | Red Hat | [arewm](https://github.com/arewm)           | May 2024 | May 2026 |
+| Brian Cook      | Red Hat | [brianwcook](https://github.com/brianwcook) | May 2024 | May 2026 |
+
+*There is no designated facilitator at the moment. The responsibility is distributed across all the*
+*members of the committee.*
+
+## Governance Facilitator Role (optional)
+
+The governance facilitator role is a non-voting, non-technical advisory position that helps ensure
+all governance committee meetings are inclusive, and key milestones toward governance are met. The
+facilitator is a role, meaning it may be occupied by a single individual, or a series of individuals
+over time. The goal of this position is servant leadership for the project, community, and
+stakeholders. The committee may choose to make this a permanent role at their discretion.
+Deliverables may include creation of mailing lists, project tracking boards, governance draft
+documents, and so forth.
+
+## Maximum Representation
+
+It is the desire of the committee to limit the amount of influence of any single employer/company over the
+project. However, currently, there is not enough diversity to make this a reality. This section
+should be revised in the future when more than one employer/company are active members of the
+Konflux community.
+
+## Committee Responsibilities and Deliverables
+
+The committee MUST:
+
+- Balance technical, architectural, and governance expertise.
+- Hold staggered terms, sufficient to ensure an orderly transition of power via elections.
+- Provide designated alternates in cases where quorum is required but not attainable with the
+  current set of members.
+
+The committee is responsible for a series of specific artifacts and activities:
+
+- The [Code of Conduct](./code_of_conduct.md) and handling violations
+- Define the processes around ADRs ([Architectural Design Records](https://adr.github.io/)) in the
+  [architecture](https://github.com/konflux-ci/architecture/tree/main/ADR) repository. Should the
+  community fail to reach consensus on whether to accept a proposed ADR or not, the governance
+  committee can help break the impasse.
+
+## Governance Decision-Making Process
+
+Governance decisions are reviewed and accepted in terms of pull requests to this repository.
+Decisions can be proposed by anyone, not just community or committee members.
+
+The governance committee decisions are taken by seeking consensus towards a motion from all its
+members. If the consensus cannot be reached, the motion may be altered or dropped.
+
+Architectural decisions are handled similarly but on the
+[architecture](https://github.com/konflux-ci/architecture) repository. The acceptance, or
+rejection, of architectural decisions abide to the same rules of the governance decision-making
+process.
```

Comment:
Does this imply the final decisions are made by the same committee? Wouldn't we want a wider forum there?

Reply:
Yes. The committee, of course, is expected to pull in whoever is needed to make such decisions.

Can you elaborate more on the desire to have a wider forum? What concerns are you trying to address?